src/test/scala-2.11/markets/clearing/engines/CDAMatchingEngineSpec.scala

```diff
@@ -260,15 +260,15 @@ class CDAMatchingEngineSpec extends TestKit(ActorSystem("CDAMatchingEngineSpec")
 
       Then("the matching engine should generate a TotalMatch at the best limit price.")
 
-      val fill = TotalMatch(marketBidOrder, askOrder, bidPrice)
+      val residualBidQuantity = bidQuantity - askQuantity
+      val (filledBidOrder, residualBidOrder) = marketBidOrder.split(residualBidQuantity)
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Field name does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://www.codacy.com/app/davidrpugh/markets-sandbox/file/1539841712/issues/source?bid=2921361&fileBranchId=3014278#l264)